### PR TITLE
Module one: capstone project

### PR DESCRIPTION
**Pull request by mulugithub**
Hello CR 🤝 

This PR, shows the capstone project

_**General requirements**_

- The home page includes a header section, a welcome section, a program section, a featured speakers section, a partner section, and a footer.
- The about page has detailed information about the company logo. 
- The section "Featured Speakers" is implemented dynamically in JavaScript.
- Responsive, both on mobile (up to 768 px wide) and desktop (768 px wide) devices.
- See more speakers
- Image transition and animation
- Mobile menu
- A live demo of the project features (different pages and different screen sizes).
- Video for project description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        </div>
+      </section>
+      <!--  end of partner section -->
+    </main>
+    <footer>
+      <a href="#"
+      ><img class="footer-logo" src="images/eta_logo.png" alt="ETA institution logo here"
+    /></a>
+      <p class="copyright">&copy; Eagle Tech Academy. Some Rights Reserved.</p>
+    </footer>
```
Comment (omarsalem7) [inline index.html:192]: - [ ] Kindly consider matching the design guidelines by making the footer content in the center and giving space evenly between elements as shown below

| Your design  | Design guidline |
| ------------- | ------------- |
| image  | image |

- [ ] Also consider the footer to have the same background as exist in the design guideline
  - In Desktop
       - About Page: It has a dark background.
       - Home Page: It has a white background.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        <article class="hero-header">
+          <p class="greeting">"Hello! Web Developers"</p>
+          <h1 class="welcome">2023 Summer Web Developer conference</h1>
+        </article>
+        <p class="about-text">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Commons affiliate network in a
+          different country every two years. This year, we hope to expand our
+          invitation list including organizations and individuals.
+        </p>
```
Comment (omarsalem7) [inline about.html:102]: - [ ] It will be better to reduce the width of this paragraph by giving it `width:70%;` to be consistent with Figma design

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @mulugithub 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Good UI design ✔️ 
- The app deployed properly using Github pages ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @mulugithub, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**